rust-toolchain.toml

```diff
@@ -2,6 +2,6 @@
 # This specifies the version of Rust we use to build.
 # Individual crates in the workspace may support a lower version, as indicated by `rust-version` field in each crate's `Cargo.toml`.
 # The version specified below, should be at least as high as the maximum `rust-version` within the workspace.
-channel = "1.79.0"
+channel = "1.81.0"
```

Comment:
Careful when updating rust minor versions without any real reason. Naivety will catch you!